lib/fluent/plugin/in_tail/position_file.rb

```diff
@@ -224,5 +263,7 @@ def read_inode
         @inode
       end
     end
+
+    PathInodeTuple = Struct.new(:path, :ino)
```

Comment:
What is the differences from `PathWithInode`?

Reply:
They are same behavior except for member names.
`PathWithInode` is not used anywhere. It should be removed the PR.

Reply:
Fixed in https://github.com/fluent/fluentd/pull/3182/commits/c1f41da80dbf9ff6289bb01031a956a3b7dc1cec.